packages/istanbul-lib-source-maps/lib/transformer.js

```diff
@@ -248,7 +253,10 @@ SourceMapTransformer.prototype.transform = function(coverageMap) {
             changed;
 
         if (!sourceMap) {
-            output[file] = fc;
+            uniqueFiles[getUniqueKey(file)] = {
```

Comment:
My memory is a bit rusty here, what is the difference between this and `getMappedCoverage`? It looks like there's duplication that can be cleaned up but I could be wrong...

cc @coreyfarrell

Reply:
The `getMappedCoverage` function is used if you have source maps and want to get coverage for the original code. If there are no source maps, just return the file coverage as is, so both are needed I guess.

Reply:
@coreyfarrell / @bcoe this is my only question left for this PR so if you feel this is fine then feel free to :shipit:

Reply:
This looks good to me.